extensions/applicationinsights-react-native/src/ReactNativePlugin.ts

```diff
@@ -48,7 +57,21 @@ export class ReactNativePlugin implements ITelemetryPlugin {
             if (!this._config.disableDeviceCollection) {
                 this._collectDeviceInfo();
             }
+            
+            if (extensions) {
+                CoreUtils.arrForEach(extensions, ext => {
+                    const identifier = (ext as ITelemetryPlugin).identifier;
+                    if (identifier === 'ApplicationInsightsAnalytics') {
+                        this._analyticsPlugin = (ext as any) as IAppInsights;
```

Comment:
Are these casts required? Or can the `as any` be removed?

Reply:
Got this error when removing `as any`: 
![image](https://user-images.githubusercontent.com/20292261/74886591-661c8400-532d-11ea-87d4-12b207659cac.png)

I think it's because ext is not necessarily analyticsPlugin? @MSNev can you please explain why we need two casts here? 🤔